.github/workflows/node-pr-jobs-secure.yml

```diff
@@ -28,10 +35,10 @@ jobs:
           echo "SERVER_MASTER_REPORT=$(cat generated/bundle-analyser/server-report.json)" >> $GITHUB_ENV 
       - name: Checkout PR Branch
         uses: actions/checkout@v2
-        # with:
-        #   repository: ${{ steps.pr_head.outputs.repo }}
-        #   ref: ${{steps.pr_head.outputs.branch}}
-        #   fetch-depth: 0
+        with:
+          repository: ${{ steps.pr_head.outputs.repo }}
+          ref: ${{steps.pr_head.outputs.branch}}
```

Comment:
https://github.community/t/running-code-from-forks-with-pull-request-target/126688/6 - does that work? If so, seems nicer than having to `jq` and export the ref.

Also, earlier in that thread it does mention risk of exporting tokens (i.e I write a script/action that can export these secrets) - not sure if that's a worry?

Reply:
Hmm good find, I'll give that a go now

The initial problem came from token security so this was supposed to work with that. Actions will mask any sensitive data and only runs because it knows it trusts the source

Reply:
That did the trick, I missed the `full_name` key which basically solves all of the problems I had 😂